Source/Extensions/Blazorise.Charts/ChartData.cs

```diff
@@ -70,20 +83,31 @@ public class ChartDataset<T>
         public int BorderWidth { get; set; } = 1;
     }
 
+    /// <remarks>
+    /// Defaults values as per https://www.chartjs.org/docs/latest/charts/line.html#dataset-properties
+    /// </remarks>
     [DataContract]
     public class LineChartDataset<T> : ChartDataset<T>
     {
+        public LineChartDataset() : base(
+            label: string.Empty,
+            backgroundColor: new List<string> { ChartColor.FromRgba(0, 0, 0, 0.1f) },
```

Comment:
add missing spaces for ChartColor.FromRgba

Reply:
Done